### ISSUE TITLE
Ask about DreaMS pre-training questions

### ISSUE THREAD

**Issue by liuzhenhuan123** [pluskal-lab/DreaMS#19]
Could you please tell me how many epochs the DreaMS pre-training process has gone through? Can you see the complete loss curve?

**Comment by roman-bushuiev** [comment]
Hi @liuzhenhuan123, sorry for the reply.

6 epochs, I am attaching loss curves. spec1 and spec2 correspond to m/z masking objectives on a pair of input spectra, RO corresponds to retention order prediction objective between these two spectra.

<img width="1409" alt="Image" src="https://github.com/user-attachments/assets/32c7a146-2e6a-4745-bcca-6a797a5dfb0b" />
